src/client/layouts/Main.tsx

```diff
@@ -1,54 +1,162 @@
+  DESKTOP: { start: 2, span: 6 },
+  LEFT_COL: { start: 3, span: 6 },
+  WIDE: { start: 4, span: 6 },
+};
+
+const headerStyles = (hasSummary = false) => css`
+  /* padding */
+  padding-top: ${space[6]}px;
+  padding-bottom: ${space[6]}px;
+
+  ${hasSummary &&
+  css`
+    padding-top: 0;
+  `}
+
+  ${from.desktop} {
+    ${hasSummary
+      ? css`
+          padding-top: 0;
+        `
+      : css`
+          padding-top: ${space[9]}px;
+        `}
+  }
+
+  /* margin */
+  margin-bottom: ${space[1]}px;
+
+  /* border */
+  border-bottom: 1px solid ${neutral[86]};
 `;
 
-const sectionStyles = css`
-  padding: ${space[6]}px ${space[3]}px;
-  max-width: ${Breakpoints.TABLET}px;
+const pageTitleStyles = css`
   width: 100%;
-  margin: 0 auto;
+  margin: 0;
+
+  ${headline.small({ fontWeight: 'bold' })}
+  font-size: 28px;
+  ${from.desktop} {
+    font-size: 32px;
+  }
+
+  color: ${text.primary};
+`;
+
+const summaryStyles = css`
+  margin: ${space[6]}px 0;
+`;
+
+const bodyStyles = (hasTitleOrSummary = false) => css`
+  ${!hasTitleOrSummary &&
+  css`
+    margin-top: ${space[1]}px;
+  `}
 `;
 
-export const Main = ({
-  subTitle,
+export const buttonStyles = ({ hasTerms = false, halfWidth = false }) => css`
+  margin-top: 22px;
+  justify-content: center;
+  width: 100%;
+
+  ${from.tablet} {
+    ${halfWidth
+      ? css`
+          width: 50%;
+        `
+      : css`
+          width: 100%;
+        `}
+  }
+
+  ${hasTerms &&
+  css`
+    margin-top: 16px;
+  `}
+`;
+
+export const MainLayout = ({
+  children,
+  pageTitle,
   successOverride,
   errorOverride,
-  children,
-}: Props) => {
+}: PropsWithChildren<MainLayoutProps>) => {
   const clientState: ClientState = useContext(ClientStateContext);
   const { globalMessage: { error, success } = {} } = clientState;
 
   const successMessage = successOverride || success;
   const errorMessage = errorOverride || error;
 
   return (
-    <main css={mainStyles}>
-      {subTitle && <SubHeader title={subTitle} />}
-      {errorMessage && (
-        <GlobalError error={errorMessage} link={getErrorLink(errorMessage)} />
-      )}
-      {successMessage && <GlobalSuccess success={successMessage} />}
-      <section css={sectionStyles}>{children}</section>
-    </main>
+    <>
+      <Header />
+      <main css={[mainStyles, gridRow]}>
+        <section css={gridItem(gridSpanDefinition)}>
+          {errorMessage && (
+            <ErrorSummary cssOverrides={summaryStyles} message={errorMessage} />
+          )}
+          {successMessage && !errorMessage && (
+            <SuccessSummary
+              cssOverrides={summaryStyles}
+              message={successMessage}
+            />
+          )}
+          {pageTitle && (
+            <header css={headerStyles(!!(errorMessage || successMessage))}>
+              <h1 css={[pageTitleStyles]}>{pageTitle}</h1>
+            </header>
+          )}
+          <div
+            css={bodyStyles(!!(pageTitle || successMessage || errorMessage))}
```

Comment:
the cases where `!!(...)` is happening for `bodyStyles`, `headerStyles` etc could maybe benefit from being moved into variables for clarity